backend/handlers/math/probabilities/wpam/wpam_marketprobabilities.go

```diff
@@ -12,6 +12,10 @@ type ProbabilityChange struct {
 	Timestamp   time.Time `json:"timestamp"`
 }
 
+type ProjectedProbability struct {
+	Probability float64 `json:"projectedprobability"`
+}
+
```

Comment:
is this type unique to wpam (i.e. not usable for dbpm)? If it is unique, this should stay here, otherwise we should consider moving it to the math or probabilities packages.